operators.js

```diff
@@ -16,6 +17,42 @@ function toBuffer(value) {
   return Buffer.isBuffer(value) ? value : Buffer.from(value)
 }
 
+function seekFromDesc(desc) {
+  return buffer => {
+    const keys = desc.split('.')
```

Comment:
This line can be moved outside inline function

Reply:
Seems to make the example query go from 15.3s to 14.5s :-)

Reply:
I can easily move it, but what do you think if we intentionally keep it slow, so that for production, developers feel motivated to have `equal` and never `slowEqual`?

Reply:
I think its good like this. Makes it easy as you say to quickly test something.